packages/bsp/common/etc/update-motd.d/30-armbian-sysinfo

```diff
@@ -53,8 +54,12 @@ function display() {
 	fi
 } # display
 
-function get_wan_address(){
-curl -s http://whatismyip.akamai.com/
+function get_wan_address() {
+    if ! /usr/bin/ping -c 1 -W 2 $CHECK_INTERNET >/dev/null 2>&1; then
+        echo "No IP Address"
```

Comment:
Or just return to not show the WAN field at all if unavailable.

Reply:
This is not the latest commit, check out this https://github.com/armbian/build/pull/7297/commits/a72b6f1aec1409a931719b437053e177bbe2ddb0